invenio.cfg

```diff
@@ -262,3 +264,16 @@ OAISERVER_ID_PREFIX = "cds-rdm.com"
 
 SEARCH_INDEX_PREFIX = "cds-rdm-"
 COMMUNITIES_ALLOW_RESTRICTED = False
+
+
+# CDS specific config
+
+AUTHORIZATION_SERVICE_API = os.environ.get("CERN_AUTHORIZATION_SERVICE_API", "https://authorization-service-api-qa.web.cern.ch/")
```

Comment:
minor: it might be overkill, but instead of using `os.environ` you can register an extension and have a prefix (e.g. CERN, CDS, etc.) and load them in the app seamlessly.